cascade/learning/chgnet.py

```diff
@@ -56,7 +56,7 @@ def evaluate(self,
                  model_msg: bytes | CHGNet,
                  atoms: list[ase.Atoms],
                  batch_size: int = 64,
-                 device: str = 'cpu') -> (np.ndarray, list[np.ndarray], np.ndarray):
```

Comment:
Nice catch! Took me a long while to realize the () syntax was wrong and tuple[] was right

Reply:
and youre the one who taught me :laughing: